encord/user_client.py

```diff
@@ -995,6 +998,171 @@ def get_client_metadata_schema(self) -> Optional[Dict[str, ClientMetadataSchemaT
+        collection_uuid_list: List[str | UUID] | None = None,
+        page_size: Optional[int] = None,
+    ) -> Iterable[Collection]:
+        """
+        Get collections by top level folder or list of collection IDs.
+        If both top_level_folder_uuid and collection_uuid_list are preset
+        then the intersection of the two conditions is returned.
+
+        Args:
+            top_level_folder_uuid: The unique identifier of the top level folder.
+            collection_uuid_list: The unique identifiers (UUIDs) of the collections to retrieve.
+            page_size (int): Number of items to return per page.  Default if not specified is 100. Maximum value is 1000.
+        Returns:
+            The list of collections which match the given criteria.
+
+        Raises:
+            :class:`encord.exceptions.AuthorizationError` : If the user does not have access to it.
+        """
+        if isinstance(top_level_folder_uuid, str):
+            top_level_folder_uuid = UUID(top_level_folder_uuid)
+        collections = (
+            [UUID(collection) if isinstance(collection, str) else collection for collection in collection_uuid_list]
+            if collection_uuid_list is not None
+            else None
+        )
+        return Collection._list_collections(
+            self._api_client,
+            top_level_folder_uuid=top_level_folder_uuid,
+            collection_uuid_list=collections,
+            page_size=page_size,
+        )
+
+    def delete_collection(self, collection_uuid: Union[str, UUID]) -> None:
+        """
+        Delete a collection by its UUID if it exists.
+
+        Args:
+            collection_uuid: The unique identifier (UUID) of the collection to delete.
+
+        Returns:
+            None
+
+        Raises:
+            :class:`encord.exceptions.AuthorizationError` : If the user does not have access to it.
+        """
+        if isinstance(collection_uuid, str):
+            collection_uuid = UUID(collection_uuid)
+        Collection._delete_collection(self._api_client, collection_uuid)
+
+    def create_collection(self, top_level_folder_uuid: Union[str, UUID], name: str, description: str = "") -> UUID:
+        """
+        Create a collection.
+
+        Args:
+            top_level_folder_uuid: The unique identifier (UUID) of the folder that the collection is created in.
+            name: The name of the collection.
+            description: The description of the collection.
+
+        Returns:
+            The UUID of the newly created collection.
+
+        Raises:
+            :class:`encord.exceptions.AuthorizationError` : If the user does not have access to the folder.
+        """
+        if isinstance(top_level_folder_uuid, str):
+            top_level_folder_uuid = UUID(top_level_folder_uuid)
+        return Collection._create_collection(self._api_client, top_level_folder_uuid, name, description)
+
+    def get_preset(self, preset_uuid: Union[str, UUID]) -> Preset:
```

Comment:
I'm worried that "Preset" is very generic word and doesn't have any connection to collection. 
So ideally it either would need to be moved in some kind of namespace, like "Collection.get_preset", or renamed to something a bit more specific and descriptive, like "CollectionPreset", "CollectionFilterPreset", or smth else.
Hope it makes sense

Reply:
Renamed to `FilterPreset`. Does that work ?